addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -63,10 +63,10 @@ const viewRegistry = registry.category("views");
 
 /**
  * @typedef {Object} ActionOptions
- * @property {Context} [additionalContext]
+ * @property {Context} [additionalContext]  This parameter will be added to the context, and remain consistent between loads.
```

Comment:
I don't know that I should actually add comments here, but I spent a truly embarassing amount of time trying to wrestle with the way the doAction method even functions.
I feel like having accessible documentation on this would help trendemously with understanding how to do complex stuff with it (or, well, any documentation at all).